yikeda6616/freee-lc
# Add interactive month selection

#8 

This pull request introduces an interactive CLI interface for selecting the target month for credit card statement retrieval. By leveraging the inquirer library, users can now choose the desired month from a list, making the process more intuitive and user-friendly.

Key Features:

Interactive selection menu for months, ranging from "This month" to "5 months ago".
Utilization of inquirer for enhanced CLI user experience.
Dynamic URL generation based on user selection to scrape the correct statement details.
This update aims to improve usability for users needing to download their credit card statements without direct API or CSV export support from their bank's website.

How to Test:

Ensure you have Node.js and npm installed.
Run npm install to install the dependencies, including inquirer.
Set your username and password in the .env file.
Run the script and follow the CLI prompts to select the desired statement month.
Verify that the correct statement CSV file is generated in the ./dist directory.
